apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/TracesResourceTest.java

```diff
@@ -349,6 +340,34 @@ void get__whenApiKeyIsPresent__thenReturnProperResponse(String apiKey, boolean e
             }
         }
 
+        @Test
+        @DisplayName("Get traces count on previous day for all workspaces, no Auth")
+        void tracesCountForWorkspace() {
+            var workspaceName = UUID.randomUUID().toString();
+            var workspaceId = UUID.randomUUID().toString();
+
+            int tracesCnt = setupTracesForWorkspace(workspaceName, workspaceId, 1);
+
+            try (var actualResponse = client.target(URL_TEMPLATE.formatted(baseURI))
+                    .path("/workspaces-traces")
+                    .request()
+                    .header(HttpHeaders.AUTHORIZATION, okApikey)
+                    .header(WORKSPACE_HEADER, workspaceName)
+                    .get()) {
+
+                assertThat(actualResponse.getStatusInfo().getStatusCode()).isEqualTo(200);
+                assertThat(actualResponse.hasEntity()).isTrue();
+
+                var response = actualResponse.readEntity(TraceCntResponse.class);
+                TraceCntResponse.WorkspaceTracesCnt workspaceTracesCnt = response.workspacesTracesCnt()
+                        .stream()
+                        .filter(wtc -> wtc.workspace().equals(workspaceId))
+                        .findFirst().orElseThrow(
+                                () -> new AssertionError("Test Workspace was not found during traces cnt request"));
+                assertThat(workspaceTracesCnt.tracesCnt()).isEqualTo(tracesCnt);
```

Comment:
You should better just compare on an expected response.

Reply:
Done